.github/workflows/forge-stable.yaml

```diff
@@ -136,7 +133,7 @@ jobs:
 
   run-forge-realistic-env-max-load-long:
     if: ${{ github.event_name != 'pull_request' && always() }}
-    needs: [ determine-test-metadata, run-forge-framework-upgrade-test ] # Only run after the previous job completes
+    needs: [determine-test-metadata, run-forge-framework-upgrade-test] # Only run after the previous job completes
```

Comment:
Somewhat unrelated, but do we need some style for yaml files?

Reply:
it's not enforced anywhere (though it probably should). I use prettier on file save

Reply:
IIRC my IntelliJ default adds the spaces 🤦

Reply:
We should use prettier everywhere.